api/docs/infra/logger.md

```diff
@@ -0,0 +1,54 @@
+# Infra
+
```

Comment:
the code and document should be in English, you know it,
JUST DO IT!!!
![image](https://github.com/marcossnikel/drawfc-api/assets/78269662/f3b01597-78ab-409f-9e38-8338f0252841)

Reply:
agree